web/src/app/monitors/monitors-list/monitors-list.component.html

```diff
@@ -16,13 +16,23 @@
     <mat-card-content>
       <mat-list>
         <mat-list-item fxLayout="column" *ngFor="let monitor of monitors">
-          <h4 mat-line><a [routerLink]="[monitor.uid,'pings']">{{monitor.name}}</a></h4>
-          <p mat-line>{{monitor.method}} {{monitor.path}} {{monitor.expectedCode}} {{monitor.expectedText}}</p>
-          <p class="button-right">
-            <button mat-raised-button color="accent" [disabled]= "manualPing" (click)="pingMonitor(monitor.uid)">Ping</button>
-            <button mat-raised-button color="accent" [routerLink]="monitor.uid">Edit</button>
-            <button mat-raised-button color="warn" (click)="deleteMonitor(monitor.uid)">Delete</button>
-          </p>
+          <div class="monitors">
+            <h4 mat-line><a [routerLink]="[monitor.uid,'pings']">{{monitor.name}}</a></h4>
+            <p mat-line>{{monitor.method}} {{monitor.path}} {{monitor.expectedCode}} {{monitor.expectedText}}</p>
+            <p mat-line>Successful pings: {{monitor.successPingCounts ? monitor.successPingCounts : 0}} | Unsuccessful pings: {{monitor.unsuccessPingCounts ? monitor.unsuccessPingCounts : 0}}  </p>
```

Comment:
I think its cleaner to do something like this...

```
monitor.successPingCounts || 0
```

Reply:
Done